src/components/JsonLink/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faRobot } from '@fortawesome/free-solid-svg-icons';
+import styles from './index.module.scss';
+
+interface Props {
+  fileName: string;
+  version: string;
+}
+
+const baseURL = `https://nodejs.org/docs/latest-`;
```

Comment:
Can this be moved to [this](https://github.com/nodejs/nodejs.dev/blob/main/apiUrls.js) file?